p2p/pkg/rpc/validator/service.go

```diff
@@ -216,3 +239,130 @@ func (s *Service) processValidators(dutiesResp *ProposerDutiesResponse) (map[uin
 
 	return validators, nil
 }
+
+func (s *Service) scheduleNotificationForSlot(epoch uint64, slot uint64, info *validatorapiv1.SlotInfo) {
+	slotStartTime := s.genesisTime.Add(time.Duration(slot) * SlotDuration)
+	notificationTime := slotStartTime.Add(-NotifyOffset)
```

Comment:
I dont think we need this delay. We already have the delay to query the duties. I think we can send this as soon as we get the info. In case of the notifications, users will be always subscribed ideally. So sooner they get it, sooner they can decide to bid.

If they miss this, they should be able to query the get validators endpoint right?